genie-web/src/integTest/java/com/netflix/genie/web/apis/rest/v3/controllers/RestControllerIntegrationTestBase.java

```diff
@@ -196,6 +196,8 @@ public void setup() throws Exception {
                         Preprocessors.modifyUris().host(URI_HOST).scheme(URI_SCHEME).removePort()
                     )
             )
+//            .addFilter(new RequestLoggingFilter())
```

Comment:
Remove these commented out lines?

Reply:
I left them in on purpose so that if we need help debugging something we don't have to continually look up what it is